scio-core/src/main/scala/com/spotify/scio/ScioContext.scala

```diff
@@ -86,7 +86,7 @@ private object RunnerContext {
 
   // FIXME: this is ugly, is there a better way?
   private def get(options: PipelineOptions): RunnerContext = {
-    val runner = options.getRunner.getName
+    val runner = Class.forName(options.getRunner.getName).getSimpleName
```

Comment:
Wait so the original returns `org.apache.beam....DirectRunner` and new one gets `DirectRunner`?
Does `options.getRunner.getSimpleName` work?

Reply:
@nevillelyh correct. And oops, yeah let me fix that.